### DISCUSSION TITLE
How to disable mouse scrolling when pressing the wheel button?

### DISCUSSION THREAD

**Discussion by John-Gee** [Alex313031/thorium#222] [Q&A] [answered]
Hey,

I am not sure how exactly this is called, but when I pressed the wheel button on something that this button cannot be used for, say on a empty part of a webpage, the whole mouse switches to scrolling mode. I would like to deactivate that, how can I?
When that happens the cursor switches to the directional arrows with a red dot in the middle. I hope that's clear enough.

Thanks!

**Comment by gz83** [comment, marked as answer]
Are you currently using the Linux version of thorium? If so, the ability to scroll a web page by pressing the mouse wheel is to fulfill a feature requested by some users. Currently this feature is not controlled by flags, but we will consider turning this feature on or off by flags, please be patient and wait for the release of the new version.

@John-Gee

**Comment by John-Gee** [comment]
Yup you guessed right I'm on Linux!
Thanks a lot for the quick reply and I look forward to the future build
allowing to control this.

**Comment by Alex313031** [comment]
@John-Gee See latest release. I disabled it by default.

**Reply by John-Gee** [reply]
Sorry, I don't restart often my browser, and finally did, that works perfectly! Thanks a lot!
